sdk/postgresql/Azure.ResourceManager.PostgreSql/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.1.0-beta.1 (2022-10-11)
```

Comment:
Please update this to a new release day as the swagger has just been merged 20 mins ago

Reply:
And please also flip the pr from `draft` to `Ready for review`